packages/forms/src/components/FilterMenu/index.tsx

```diff
@@ -141,7 +144,7 @@ export default function FilterMenu({
             <T k="lunar.common.apply" phrase="Apply" context="Apply filters button label" />
           </Link>
 
-          <Link muted type="reset" onClick={handleClear}>
+          <Link muted={!unmutedReset} type="reset" onClick={handleClear}>
```

Comment:
ah - chatted offline. since the concern is that the muted is too light/looks disabled (agreed), let's replace Link with SecondaryLink, remove muted. this'll make the link darker and match FormActions pattern better (primary color for main action like Submit/Apply, secondary color of Cancel/Clear). no need for a new prop then. better consistency!